layouts/shortcodes/copytable.html

```diff
@@ -0,0 +1,69 @@
+<style>
+.copy-btn {
```

Comment:
The site has a dark and light theme. Could we theme the button so it changes when dark theme is used?  When toggled the `html` tag goes from:

```html
<html lang="en" style="color-scheme: light;" class="">
```

to this:

```html
<html lang="en" style="color-scheme: dark;" class="dark">
```

Maybe model the color scheme (including `:hover` color)  off of the `Did this documentation help you ?` at the bottom? Though...that looks only mediocre.  You pick the changes!  We can always revise at a later date as needed.

Reply:
screenshots look great - thank you!